src/inet/tests/TestInetEndPoint.cpp

```diff
@@ -172,6 +178,21 @@ static void TestInetInterface(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, !intIterator.SupportsMulticast());
     NL_TEST_ASSERT(inSuite, !intIterator.HasBroadcastAddress());
 
+    // Not all platforms support getting interface type and hardware address
+    err = intIterator.GetInterfaceType(intType);
```

Comment:
If I'm not misreading, by this point the iterator has run off the end of the loop above, so the `GetInterfaceType()` and `GetHardwareAddress()` shouldn't succeed.

Reply:
Ah yes, you are right, I misunderstood the test purpose. Thanks for spotting that, I fixed that.